runtime/kusama/src/lib.rs

```diff
@@ -610,13 +581,11 @@ construct_runtime! {
 		Slots: slots::{Module, Call, Storage, Event<T>},
 		Registrar: registrar::{Module, Call, Storage, Event, Config<T>},
 
-		// Simple nicknames module.
-		// KUSAMA: Remove before mainnet
-		Nicks: nicks::{Module, Call, Storage, Event<T>},
```

Comment:
Why is nicks removed?

Reply:
Was scheduled to be removed (identity replaces it)